packages/@react-aria/focus/src/FocusScope.tsx

```diff
@@ -147,7 +147,8 @@ function getFocusableElementsInScope(scope: HTMLElement[], opts: FocusManagerOpt
     }
     res.push(...Array.from(node.querySelectorAll(selector)));
   }
-  return res;
+  // Filter out elements that have display: none or whose parents have display: none. https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/offsetParent
+  return res.filter(node => node.offsetParent !== null);
```

Comment:
Feel like this solution is pretty iffy tbh since it applies to `display: none` only and it will filter out elements that are visible but have `position: fixed` (dunno if we'll have a case where FocusScope wraps a element that has fixed position).

Alternate solution was to get the element's computed style to check if `visibility: hidden` or `display: none` (doesn't work if the parent's style are hiding the children). Maybe I could check the size of the element, dunno